pipeline/README.md

```diff
@@ -68,14 +67,40 @@ How to Run the Pipeline
 
 2. **Set Up Environment Variables**:
     Use a `.env` file to provide database credentials and schema information:
-    DB_HOST=<your_database_host> DB_NAME=<your_database_name> DB_USER=<your_database_user> DB_PASSWORD=<your_database_password> DB_PORT=<your_database_port> SCHEMA_NAME=<your_schema_name>
+    DB_HOST=<your_database_host>
+    DB_NAME=<your_database_name>
+    DB_USER=<your_database_user>
+    DB_PASSWORD=<your_database_password>
+    DB_PORT=<your_database_port>
+    SCHEMA_NAME=<your_schema_name>
 
 3. **Run the Pipeline**:
     Execute the pipeline by running:
     ```bash
     python3 pipeline.py
     ```
 
+Docker Deployment
+-------
+The ETL pipeline can also be run as a Docker container using the minute_pipeline_dockerfile.
+
+**Building the Docker Image**
+    Navigate to the project root directory and run the following command:
+    ```bash
+    docker build --file pipeline/minute_pipeline_dockerfile -t team_growth_pipeline:v1.0 . --platform "linux/amd64"
```

Comment:
Consider explaining why --platform "linux/amd64" is specified in the docker build command. This would be helpful for users who may be unfamiliar with platform compatibility.